FEM/rf_react_int.cpp

```diff
@@ -1,3187 +1,3187 @@
-    liquid.volume = liquid.mass / liquid.density;
-
-    vapor.temperature = T;
-    vapor.pressure = P;
-    vapor.density = VLE::density_CO2(T, P);  // to use mixture fluid EoS
-    vapor.viscosity = -1.0;
-    vapor.mass = vapor.CO2 * 44.009 + vapor.H2O * 18.01528;
-    vapor.volume = vapor.mass / vapor.density;
-}
-
-void VLE_isochoric_fixphase(double T, double& P, Phase_Properties2& vapor,
-                            Phase_Properties2& liquid, Phase_Properties2& solid,
-                            int f)
-{
-    int i = 0, i_max = 31;
-    double V = 0, V0 = 0, P1 = 0, P2 = 0;
-    double err = 1.0e-8;
-
-    V0 = vapor.volume + liquid.volume;
-    VLE_isobaric_fixphase(T, P, vapor, liquid, solid, f);
-    V = vapor.volume + liquid.volume;
-
-    if (abs(V - V0) < err)
-        return;
-    else if (V > V0)
-    {
-        P1 = P;
-        P2 = 5.0 * P;
-    }
-    else if (V < V0)
-    {
-        P1 = 0.2 * P;
-        P2 = P;
-    }
-    for (i = 0; i < i_max; i++)
-    {
-        P = 0.5 * (P1 + P2);
-        VLE_isobaric_fixphase(T, P, vapor, liquid, solid, f);
-        V = vapor.volume + liquid.volume;
-        if (abs(V0 - V) < err)
-            break;
-        else if (V < V0)
-            P2 = P;
-        else if (V > V0)
-            P1 = P;
-    }
-    if (i == i_max)
-    {
-        cout << "VLE_isochoric_fixphase() max.it.=" << i
-             << " reached, dV=" << abs(V - V0) << " rho=";
-        cout << vapor.density * 1000 << "\n";
-    }
-}
+/**
+ * \copyright
+ * Copyright (c) 2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+/*
+  rf_react_int.cpp
+  Reaction interface
+*/
+
+#include "rf_react_int.h"
+#include "display.h"
+#include "files0.h"
+#include "makros.h"
+#include "mathlib.h"
+#include "rf_ic_new.h"
+#include "rf_kinreact.h"
+#include "rf_mmp_new.h"
+#include "rf_pcs.h"
+//#include "rf_react.h"
+#include <signal.h>
+#include <cfloat>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <vector>
+#include "rf_tim_new.h"
+#include "rfmat_cp.h"
+#include "stdio.h"
+#include "tools.h"
+
+// Elem object
+#include "fem_ele_std.h"
+
+#include "Density.h"
+#include "VLE.h"
+
+#include <vector>
+using namespace std;
+using namespace Display;
+
+extern double gravity_constant;
+
+vector<REACTINT*> REACTINT_vec;
+
+/**************************************************************************/
+// REACINT class
+/**************************************************************************/
+
+/**************************************************************************
+ ReacInt-Method:
+ Task: Constructor
+ Programing:
+ //CB 01.2011 CB First implementation
+ **************************************************************************/
+REACTINT::REACTINT(void)
+{
+    unitconversion = false;
```

Comment:
This file is reformatted. Please use the original file and only add new changes.
You can use `git-clang-format HEAD~1` command for format the files once the files are added by git add.